solved_exercises/dani/02-tuples.py

```diff
@@ -17,8 +19,9 @@
 
 '************************************************'
 print('Find the repeated items of a tuple')
-my_tuple = 1, 2, 3, 3, 3, 5, 5, 6
-my_multiples = [item for item in my_tuple if my_tuple.count(item)>1]
+my_tuple = (1, 2, 3, 3, 3, 5, 5, 6)
+my_multiples = (item for item in my_tuple if my_tuple.count(item)>1)
+my_multiples = list(dict.fromkeys(my_multiples))
```

Comment:
there is no need to go through a dictionary

Reply:
I have no idea why I did that :)